README.md

```diff
@@ -1,239 +1,126 @@
 
-<p align="center">
-  A scalable stateless Authorization Service for Federated Identities including Google and Facebook
-</p>
+<br />
 
-<p align="center">
-  <a href="http://www.overture.bio/products/ego" target="_blank"><img alt="General Availability" title="General Availability" src="http://www.overture.bio/img/progress-horizontal-GA.svg" width="320" /></a>
-</p>
+<p align="center"><a href="https://www.overture.bio/" target="_blank"><img src="Overture_logo.png" width="66%"></a></p>
 
-[![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=U3dLZnRFNWI2MWNFY2NGcXVtVTB3WDcyU2dPVjlVeEFYUEdxUnpYZlhrUT0tLTFzY0taYTA0MVFEa3ErNkRZdTBRWVE9PQ==--690f89a41a0eedf7b4975bd7df2eac162e04e775% )](https://www.browserstack.com/automate/public-build/U3dLZnRFNWI2MWNFY2NGcXVtVTB3WDcyU2dPVjlVeEFYUEdxUnpYZlhrUT0tLTFzY0taYTA0MVFEa3ErNkRZdTBRWVE9PQ==--690f89a41a0eedf7b4975bd7df2eac162e04e775%)
-[![Slack](http://slack.overture.bio/badge.svg)](http://slack.overture.bio)
+<br />
 
-## Table of Contents
-- [Documentation](#documentation)
-- [Introduction](#introduction)
-- [Features](#features)
-- [Tech Stack](#tech-stack)
-- [Quick Start](#quick-start)
-- [Development Install](#development-install)
-  - [Step 1 - Setup Database](#step-1---setup-database)
-  - [Step 2 - Run](#step-2---run)
-- [Tech Specifications](#tech-specification)
-- [Usage](#usage)
-- [Shoutouts](#shoutouts)
-  - [Browserstack](#browserstack)
-
-## Documentation
-
-Explore documentation with the Ego [Read the Docs](https://ego.readthedocs.io/en/latest/).
+<!-- Replace slack with discourse once setup -->
 
-## Introduction
+[<img src="https://img.shields.io/badge/chat-on--slack-blue">](http://slack.overture.bio) 
+[<img src="https://img.shields.io/badge/License-gpl--v3.0-blue">](https://github.com/overture-stack/ego/blob/develop/LICENSE)
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+## Worry Less Science More
 
-Authorization Service built to provide Single Sign On for various microservices in an application. EGO works with Identity Providers such as Google, Facebook to provide social logins in the application. EGO provides stateless authorization using [JWT (JSON Web Tokens)](https://jwt.io/) and can scale very well to a large number of users.
+In this Overture repository, we (the [OICR Genome Informatics Team](https://softeng.oicr.on.ca/team/)) develop [Ego](https://www.overture.bio/products/ego/), our open-source authentication and authorization microservice. [Overture](https://www.overture.bio/) is an ensemble of modular solutions for big-data genomic science. Our core products work in concert to manage, explore and visualize molecular and clinical data. Visit our [website](https://www.overture.bio/) for more information on what Overture offers, and check out our other projects on [GitHub](https://github.com/overture-stack/).
```

Comment:
No need to declare `Overture repository`, we already have github indicating that `overture-stack` is the organization this repository belongs in.

The first paragraph should focus on details of the ego application and not the developers.

I would rearrange the first two sections you have to put the Ego section at the top, then move the Overture section beneath. The overture section perhaps could have a standard re-usable paragraph about what overture is, then a custom paragraph about how Ego fits into the Overture application suite.

Something like:


# Ego

## Secure Authentication & Authorization
Ego is an open-source authentication and authorization microservice, providing secure user registration and permission management. Ego uses well-known single-sign-on identity providers like Google, GitHub, LinkedIn and ORCiD in place of managing usernames and passwords. 

## Overture - Worry Less, Science More
< overutre logo here >
[Overture](https://www.overture.bio/) is an ensemble of modular solutions for big-data genomics science. Our core products work in concert to manage, explore and visualize molecular and clinical data. Visit our [website](https://www.overture.bio/) for more information on what Overture offers, and check out our other projects on [GitHub](https://github.com/overture-stack/).

Reply:
Addressed!